src/main/scala/jigg/util/JSONUtil.scala

```diff
@@ -35,13 +42,13 @@ object JSONUtil {
     def hasAttribution(x: Node): Boolean = !x.attributes.isEmpty
   }
 
-  def toJSON(x: Any): StringBuilder = x match {
+  def toJSON(x: Any): String = x match {
```

Comment:
I think it's better to segment this method into different methods for each argument type, e.g., `readJSON(file: String)` (or `file: File`) and `toJSON(node: Node)` (or `node: Elem`?). Generally a method receiving `Any` type and changing the behavior internally is not a good practice.